src/main/java/bh/bot/app/ChangeCharacterApp.java

```diff
@@ -0,0 +1,142 @@
+            info(getHelp());
+            arg = readInputLoopCount("What character slot do you want to pick?");
+        }
+
+        final int characterSlot = arg;
+        info("Character in slot %3d ", characterSlot);
+        AtomicBoolean masterSwitch = new AtomicBoolean(false);
+        ThreadUtil.waitDone(
+                () -> doLoopClickImage(characterSlot, masterSwitch),
+                () -> internalDoSmallTasks( //
+                        masterSwitch, //
+                        SmallTasks //
+                                .builder() //
+                                .clickTalk() //
+                                .clickDisconnect() //
+                                .reactiveAuto() //
+                                .autoExit() //
+                                .detectChatboxDirectMessage() //
+                                .build() //
+                ), //
+                () -> doCheckGameScreenOffset(masterSwitch));
+        Telegram.sendMessage("Stopped", false);
+    }
+
+    private void doLoopClickImage(int characterSlot, AtomicBoolean masterSwitch) {
+        info(ColorizeUtil.formatInfo, "\n\nStarting Character Change");
+        Main.warningSupport();
+        try {
+            final int mainLoopInterval = Configuration.Interval.Loop.getMainLoopInterval(getDefaultMainLoopInterval());
+
+            moveCursor(new Offset(100, 500).toScreenCoordinate());
+            boolean characterLoaded = false;
+            boolean loadedSelect = false;
+            boolean characterSelected = false;
+            int loopCount = 0;
+            while (!characterLoaded && !masterSwitch.get()) {
+                sleep(mainLoopInterval);
+                if (clickImage(BwMatrixMeta.Metas.Character.Buttons.characterSelect)) {
+                    debug("Loading Character Selection");
+                }
+                if (characterSelected) {
+                    loopCount += 1;
+                    if (loopCount > 10) {
+                        debug("We probably loaded the character, or already on it, so breaking out of the loop!");
+                        InteractionUtil.Keyboard.sendEscape();
+                        InteractionUtil.Keyboard.sendEscape();
+                        break;
+                    }
+                }
+                if (loadedSelect) {
+                    if (characterSelected) {
+                        sleep(mainLoopInterval);
+                        if (clickImage(BwMatrixMeta.Metas.Character.Dialogs.loading)) {
+                            debug("Character Loading!");
+                            characterLoaded = true;
+                            break;
+                        }
+                    } else {
+                        debug("Selecting Character in Slot #" + characterSlot);
+                        if (characterSlot == 1) {
+                            InteractionUtil.Mouse.mouseMoveAndClickAndHide(new Offset(250, 200).toScreenCoordinate());
```

Comment:
hardcoded offset

Reply:
These are hardcoded as there is no unique image in that location that can be recognized.

Reply:
The main purpose of Profile design is too support multiple screen resolution